Makefile

```diff
@@ -94,6 +94,9 @@ dev:
 	curl -sSLO https://raw.githubusercontent.com/helm/chart-testing/v$(ct_version)/etc/lintconf.yaml; \
 	cd -;
 	echo "$(kelik) Imported ct config";
+	echo "=> Installing standard ESLint config..."; \
```

Comment:
Why do we need to support `make dev`? We will always have problems with this code over time.
We can just have a list in README.md:
```
- helm
- helm-docs
- chart-testing
```
that all, devs will install this software as they like(no need to think about brew, etc.).

Reply:
Having a single command for configuring the development environment, reduces the toil and ensures that there's only one right way to configure the development environment.